### DISCUSSION TITLE
Why does the jest unit test of swiper v8.0.3 report an error Cannot find module 'swiper/react' from 'components/AppList/index.tsx'

### DISCUSSION THREAD

**Discussion by holo0906** [nolimits4web/swiper#5451] [Q&A]
![image](https://user-images.githubusercontent.com/54050545/153384751-f98205a5-58ba-4c3e-bd9c-f36aa9491547.png)
![image](https://user-images.githubusercontent.com/54050545/153385694-5aa22100-aa90-4c10-9966-4ae4eda0dd83.png)

**Comment by nolimits4web** [comment]
because of this https://github.com/facebook/jest/issues/9771

**Comment by suren-tavant** [comment]
I'm facing the same issue.

![image](https://user-images.githubusercontent.com/99355972/169784270-8c510ac7-9257-49ed-a7a8-49ac5a40d0cd.png)

My current version is: **swiper**: "^8.1.4",
